MonoGame.Framework/Graphics/Effect/BasicEffect.cs

```diff
@@ -192,7 +184,9 @@ public float Alpha
             }
         }
 
-        /// <inheritdoc/>
```

Comment:
Note `<inheritdoc/>` was here on purpose.

The docs for some of these methods are already documented in the interface.  In this case:

https://github.com/mono/MonoGame/blob/develop/MonoGame.Framework/Graphics/Effect/IEffectLights.cs

... we do not want duplicate documentation.